CONTRIBUTING.md

```diff
@@ -38,17 +46,89 @@ OS            | Supported tests
 OS X          | Android, iOS
 Windows       | Android
 
+### Test descriptions
+
+The tests first build the app.
+
+They then check if the required emulators are currently running.
+
+If an Android emulator is not running, it attempts to boot an Android emulator named ```emulator```. You can specify an emulator by adding ```--androidemu yourEmulatorNameHere``` as a command line option to the gulp task.
+
+If an iOS simulator is not running, it attempts to boot the latest iOS iPhone simulator. You can specify a simulator by adding ```--iosemu yourSimulatorNameHere``` as a command line option to the gulp task.
```

Comment:
can't choose target for both because you can run both android and ios tests with one command. I think we might as well leave it like this for consistency and also because the emulator boot script calls the iOS simulator an emulator (so I could reuse the code easily)